torch/_dynamo/config.py

```diff
@@ -58,6 +50,8 @@
     torch._utils.is_compiling: True,
 }
 
+# Here for bw compat, will be removed (mlazos)
```

Comment:
Why don't you tell people in comments how to use the new system to do what the old comment did? I know at least I still pop config.py open and edit it to modify logging.

Reply:
still valid

Reply:
I put it above by log_level, I can point them up there